### DISCUSSION TITLE
Incorrect 'manufacturerId' for Zooz S2 ZST10 700-series controller?

### DISCUSSION THREAD

**Discussion by nop-ad-infinitum** [zwave-js/zwave-js#2318] [Q&A] [answered]
Hey. First, thanks to everyone who's contributing to this project. As an independent developer, zwave-js is an awesome resource and I'm very grateful.

I'm posting because I seem to have a problem with my new Zooz 700-series controller. It works normally, so far as I've tested it (including/excluding nodes, issuing commands to nodes on the network, network healing, hard resets, etc.). However, for some reason it doesn't give me the correct value for the stick's 'manufacturerId' property, which always reads "0". I have almost no experience with Z-Wave, so probably I'm doing something wrong, but my two older ZST10 S2 sticks (not 700-series) return the id for Zooz as expected. Could the lack of a device-specific config file be the issue? I've reached out to Zooz to see if they can help, but figured I'd ask here in case anyone recognized my problem.

Thanks!

**Comment by AlCalzone** [comment, marked as answer]
This is correct. So far, the new 700-series sticks all share the same firmware which is provided as a binary from Silabs. So they also share the same manufacturerId.

**Reply by nop-ad-infinitum** [reply]
Fantastic. Thanks for clearing that up for me!
